src/index.html

```diff
@@ -22,6 +22,468 @@
+      >
+        <img
+          src="./images/logo.png"
+          alt="Moyo logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__links">
+          <li class="nav__link-item">
+            <a
+              class="nav__link is-active"
+              href="/apple"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li class="nav__link-item">
+            <a
+              class="nav__link"
+              href="/samsung"
+            >
+              Samsung
+            </a>
+          </li>
+
+          <li class="nav__link-item">
+            <a
+              class="nav__link"
+              href="smartphones"
+            >
+              Smartphones
+            </a>
+          </li>
+
+          <li class="nav__link-item">
+            <a
+              data-qa="nav-hover"
+              class="nav__link"
+              href="/laptops"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__link-item">
+            <a
+              class="nav__link"
+              href="/gadgets"
+            >
+              Gadgets
+            </a>
+          </li>
+
+          <li class="nav__link-item">
+            <a
+              class="nav__link"
+              href="/tablets"
+            >
+              Tablets
+            </a>
+          </li>
+
+          <li class="nav__link-item">
+            <a
+              class="nav__link"
+              href="/photo"
+            >
+              Photo
+            </a>
+          </li>
+
+          <li class="nav__link-item">
+            <a
+              class="nav__link"
+              href="/video"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="products">
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <img
+          class="card__image"
+          src="./images/imac.jpeg"
+          alt="Product"
+        />
+
+        <div class="card__content">
+          <div class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </div>
+
+          <div class="card__product-code">Product code: 195434</div>
+
+          <div class="card__reviews">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+
+              <div class="stars__star"></div>
+
+              <div class="stars__star"></div>
+
+              <div class="stars__star"></div>
+
+              <div class="stars__star"></div>
+            </div>
+            <div class="card__reviews-count">Reviews: 5</div>
+          </div>
+
+          <div class="card__price">
+            <span class="card__price-text">Price</span>
+
+            <span class="card__price-value">$2,199</span>
+          </div>
+
+          <a
+            data-qa="card-hover"
+            href="#"
+            class="card__button"
+          >
```

Comment:
The button should have a `type="button"` if it is not intended to submit a form, to ensure correct behavior across all browsers.